resources/views/contests/index.blade.php

```diff
@@ -31,10 +31,12 @@
                             </tr>
                         </thead>
                         <tbody>
+
+
                             @foreach($contests as $contest)
                                 <tr class="table__row">
                                     <td class="table__cell"><a href="{{ route('contests.show', $contest->id) }}">{{ $contest->id }}</a></td>
-                                    <td class="table__cell">{{ $contest->campaign_id }}</td>
+                                    <td class="table__cell"> {{ link_to(config('services.phoenix.uri') .'/node/' . $contest->campaign_id, $contest->campaign_id) }} </td>
```

Comment:
The `link_to` helper function returns the link HTML as a string, like `<a href="#">123</a>`. So I haven't pulled this down to test, but I think the `{{` tags would be escaping those into `&lt;a href=&quot;#&quot;&gt;123&lt;/a&gt;`.